comid/keytriple.go

```diff
@@ -1,21 +1,21 @@
-// Copyright 2021-2023 Contributors to the Veraison project.
+// Copyright 2021-2024 Contributors to the Veraison project.
 // SPDX-License-Identifier: Apache-2.0
 
 package comid
 
 import "fmt"
 
-// DevIdentityKey stores an identity-triple-record with CBOR and JSON
+// KeyTriple stores an identity-triple-record with CBOR and JSON
```

Comment:
Is line 8 a fair statement, now that KeyTriple is a common type for both Identity and Attestation Key ?

Reply:
Good point. Updated to "KeyTriple stores a cryptographic key triple record (identity-triple-record
or attest-key-triple-record) with CBOR and JSON"